docker/flags.go

```diff
@@ -48,6 +48,7 @@ var (
 		{"login", "Register or log in to a Docker registry server"},
 		{"logout", "Log out from a Docker registry server"},
 		{"logs", "Fetch the logs of a container"},
+		{"network", "Manages docker network"},
```

Comment:
I think you can create a "flags_experimental.go" file in the "docker" directory and add the "// +build experimental" build tag at the top of the file.
Then you can add the network command in a "init" function in the new file.
For example:
```diff
diff --git a/docker/flags_experimental.go b/docker/flags_experimental.go
new file mode 100644
index 0000000..3b7f32d
--- /dev/null
+++ b/docker/flags_experimental.go
@@ -0,0 +1,7 @@
+// +build experimental
+
+package main
+
+func init() {
+	dockerCommands = append(dockerCommands, command{"network", "Network management"})
+}
```